commands/clients.go

```diff
@@ -132,6 +135,28 @@ func setServerTransport(u *url.URL, insecureTLS bool, l *logrus.Entry) {
 	managementpb.Default.SetTransport(transport)
 }
 
+// ParseCustomLabels parses --custom-labels flag value.
+//
+// Note that quotes around value are parsed and removed by shell before this function is called.
+// E.g. the value of [[--custom-labels='region=us-east1, mylabel=mylab-22']] will be received by this function
+// as [[region=us-east1, mylabel=mylab-22]].
+func ParseCustomLabels(labels string) (map[string]string, error) {
+	result := map[string]string{}
```

Comment:
This project for some reason hasn't `go-consistent` linter, however, I remember that we use `new`  for maps initialization.

Reply:
copy-pasted from https://github.com/percona/pmm-admin/blob/master/commands/base.go#L130 :)